<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>10</y>
      <width>729</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="browser">
     <attribute name="title">
      <string>Browser</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>721</width>
        <height>531</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="browserLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2">
         <item>
          <widget class="QComboBox" name="depBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="chrBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>[+] Status</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="editor">
     <attribute name="title">
      <string>Editor</string>
     </attribute>
     <widget class="QLabel" name="statusLabelEditor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>[+] Status</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>115</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>501</width>
        <height>305</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QComboBox" name="depEditorBox"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="depKeyLabel">
         <property name="text">
          <string>Key</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="depKeyEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="depNameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="depNameEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="depFNameLabel">
         <property name="text">
          <string>Dean's first name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="depFNameEdit"/>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="debLNameLabel">
         <property name="text">
          <string>Dean's last name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="depLNameEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QComboBox" name="chrEditorBox"/>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="chrCodeLabel">
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="chrCodeEdit"/>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="chrNameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="chrNameEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QComboBox" name="profEditorBox"/>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="profFNameLabel">
         <property name="text">
          <string>Professor's first name</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="profFNameEdit"/>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="profLNameLabel">
         <property name="text">
          <string>Professor's last name</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="profLNameEdit"/>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="profRecLabel">
         <property name="text">
          <string>Record</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="profRecEdit"/>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="profSexLabel">
         <property name="text">
          <string>Sex</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLineEdit" name="profSexEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBrowser">
    <property name="title">
     <string>Browser</string>
    </property>
   </widget>
   <addaction name="menuBrowser"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
